SonarQube.Bootstrapper/BuildAgentUpdater.cs

```diff
@@ -108,7 +114,7 @@ private static string GetDownloadZipUrl(string url)
                 downloadZipUrl = downloadZipUrl.Substring(0, downloadZipUrl.Length - 1);
             }
 
-            downloadZipUrl = string.Format(System.Globalization.CultureInfo.InvariantCulture, "{0}{1}", BootstrapperSettings.IntegrationUrlSuffix, downloadZipUrl);
+            downloadZipUrl = string.Format(System.Globalization.CultureInfo.InvariantCulture, "{0}{1}", downloadZipUrl, BootstrapperSettings.IntegrationUrlSuffix);
```

Comment:
I changed the way this string was constructed in the previous commit and managed to get it wrong.
I've added a unit test.